ci/benchmark/build.sh

```diff
@@ -77,11 +77,16 @@ conda install "rmm=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
 # conda remove -f rapids-build-env rapids-notebook-env
 # conda install "your-pkg=1.0.0"
 
-# Install the master version of dask, distributed, and streamz
-logger "pip install git+https://github.com/dask/distributed.git@$DASK_DISTRIBUTED_GIT_TAG --upgrade --no-deps"
-pip install "git+https://github.com/dask/distributed.git@$DASK_DISTRIBUTED_GIT_TAG" --upgrade --no-deps
-logger "pip install git+https://github.com/dask/dask.git@$DASK_DISTRIBUTED_GIT_TAG --upgrade --no-deps"
-pip install "git+https://github.com/dask/dask.git@$DASK_DISTRIBUTED_GIT_TAG" --upgrade --no-deps
+# Install the conda-forge or nightly version of dask and distributed
+if [[ "${INSTALL_DASK_MAIN}" == 1 ]]; then
+    gpuci_logger "gpuci_mamba_retry update dask"
+    gpuci_mamba_retry update dask
+else
+    gpuci_logger "gpuci_mamba_retry install conda-forge::dask>=2021.11.1 --force-reinstall"
+    gpuci_mamba_retry install conda-forge::dask>=2021.11.1 --force-reinstall
```

Comment:
```suggestion
    gpuci_mamba_retry install "conda-forge::dask>=2021.11.1" --force-reinstall
```

Reply:
It seems to be changing `dask` package alone and not `distributed` & `dask-core`:
```


  Package                 Version  Build              Channel                       Size
──────────────────────────────────────────────────────────────────────────────────────────
  Downgrade:
──────────────────────────────────────────────────────────────────────────────────────────

  - dask         2022.02.1a220223  py_g8a99ac7c_31    installed                         
  + dask                 2022.2.0  pyhd8ed1ab_0       conda-forge/noarch            5 KB
  - dask-core    2022.02.1a220223  py_g61db08d2_28    installed                         
  + dask-core    2022.02.0a220216  py38_g47fa383c_12  dask/label/dev/noarch       801 KB
  - distributed  2022.02.1a220223  py39_g8a99ac7c_31  installed                         
  + distributed  2022.02.0a220216  py39_g94d62268_15  dask/label/dev/linux-64       1 MB

  Summary:

  Downgrade: 3 packages

  Total download: 2 MB
```

Should we specify all the three packages at this step?

Reply:
Yeah I think we're going to need to do that